src/System.Text.Http.Parser/HttpParser.cs

```diff
@@ -446,7 +446,7 @@ public HttpParser(bool showErrorDetails)
                                 return false;
                             }
 
-                            var headerSpan = buffer.Slice(index, end - index + 1).ToSpan();
+                            var headerSpan = Sequence.ToSpan(buffer.Slice(index, end - index + 1));
```

Comment:
I don't think this is right. There was another ToSpan

Reply:
Ah yes, https://github.com/dotnet/corefxlab/blob/71ea1a077c6852a2869909117828c66822051b74/src/System.Text.Http.Parser/PipelineExtensions.cs#L12.

I will update this to call the right ToSpan method. Do we need to update it anywhere else?

Other places use: https://github.com/dotnet/corefxlab/blob/71ea1a077c6852a2869909117828c66822051b74/src/System.Buffers.Experimental/System/Buffers/Sequences/SequenceExtensions.cs#L12